firebase/Registration.ts

```diff
@@ -47,7 +47,8 @@ export async function storeInitialUserData(uid: string, email: string, firstName
       location: location,
       description: description,
       selectedInterests: selectedInterests,
-      profilePicture: ""
+      profilePicture: "",
+      events : []
```

Comment:
I think that we don't want this, because the context doesn't allow for it. Are you sure we need it?

Reply:
We need it to avoid an error, fetching null events. Now we can differentiate if it's null or just empty